model.py

```diff
@@ -1420,23 +1420,23 @@ def url(self):
         return self.setting(self.URL).value
 
     @url.setter
-    def set_url(self, new_url):
+    def url(self, new_url):
```

Comment:
Following a change in SQLAlchemy, the setter method for a hybrid property needs to have the same name as the getter method.